src/sequence_processing_pipeline/Pipeline.py

```diff
@@ -28,7 +28,7 @@
 class InstrumentUtils():
     types = {'A': 'NovaSeq 6000', 'D': 'HiSeq 2500', 'FS': 'iSeq',
              'K': 'HiSeq 4000', 'LH': 'NovaSeq X Plus', 'M': 'MiSeq',
-             'MN': 'MiniSeq',
+             'MN': 'MiniSeq', 'SH': 'MiSeq i100',
```

Comment:
I am worried about this one, not for this code, but for `kl-metapool`, which AFAICT doesn't know about MiSeq i100 (as opposed to just regular MiSeq).  What do we know about whether it needs its i5 barcode revcomped or not, etc?  Is `kl-metapool` going to need modifications to handle this sequencer?